cmd/root.go

```diff
@@ -31,28 +33,59 @@ import (
 )
 
 var (
-	cfgFile                string
-	cpuprofile, memprofile string
+	cfgFile                        string
+	cpuprofile, memprofile         string
+	cpuprofileFile, memprofileFile *os.File
 )
 
 // rootCmd represents the base command when called without any subcommands
 var rootCmd = &cobra.Command{
-	Use:     "hubble",
-	Short:   "CLI",
-	Long:    `Hubble is a utility to observe and inspect recent Cilium routed traffic in a cluster.`,
-	Version: pkg.Version,
+	Use:          "hubble",
+	Short:        "CLI",
+	Long:         `Hubble is a utility to observe and inspect recent Cilium routed traffic in a cluster.`,
+	SilenceUsage: true,
+	Version:      pkg.Version,
+	PersistentPreRunE: func(_ *cobra.Command, _ []string) error {
```

Comment:
maybe move the inners to a separate function - the command definition will the remain cleaner